src/tests/Register.test.js

```diff
@@ -106,4 +107,80 @@ it('blocks the user to register without password confirmation', async () => {
 
     expect(screen.queryByLabelText("response")).not.toBeInTheDocument();
     
-})
\ No newline at end of file
+})
+
+it('checks empty field warning', async () => {
+
+    render(<Register />)
+
+
+    fireEvent.change(screen.getByLabelText("Name :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
+
+    fireEvent.change(screen.getByLabelText("Username :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
+    fireEvent.change(screen.getByLabelText("Email :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
+    fireEvent.change(screen.getByLabelText("Password :", { selector: "input" }), {
+        target: { value: '' },
+    })
+    
+    fireEvent.change(screen.getByLabelText("Confirm Password :", { selector: "input" }), {
+        target: { value: '' },
+    })
+
+    act(() => {
+        fireEvent.click(screen.getByRole('button', { name: "Sign Up" }), {
+            target: { value: 'true' },
+        })
+    });
+    expect(screen.getByLabelText("Name :", { selector: "input" })).toBeRequired();
```

Comment:
@Aaishpra , I just realized that these assert lines (143-147) need to be wrapped with the `await wait(()` to clear the warrning message (just like what you've done in PR #95 . Can you please fix this?

Reply:
Sure

Reply:
Great, @Aaishpra , just one minor thing. Indent the assert lines, please (here as well as on lines 186-190). Once you done them, I'll approve asap. 😉

Reply:
> Great, @Aaishpra , just one minor thing. Indent the assert lines, please (here as well as on lines 186-190). Once you done them, I'll approve asap. wink

Is this fine now @mtreacy002 ?